packages/iptables/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -378,4 +379,4 @@ on_failure:
       value: pipeline_error
   - append:
       field: error.message
-      value: '{{{ _ingest.on_failure_message }}}'
+      value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag {{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
```

```suggestion
     value: >-
       Processor '{{ _ingest.on_failure_processor_type }}'
       {{#_ingest.on_failure_processor_tag}}with tag '{{ _ingest.on_failure_processor_tag }}'
       {{/_ingest.on_failure_processor_tag}}failed with message '{{ _ingest.on_failure_message }}'
```

(that's my personal preference for this message b/c it omits the tag when it's not present)

---

tbh I did borrow that value from another integration (which I don't recall atm) but not a prob I like `>-` approach as well

---

I think that now this is a nicer format. Please feel free to unresolve if you spot anything else